### ISSUE TITLE
b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019b

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#149]
**Description:**
Same as 017 (Issue: https://github.com/NCAR/amwg_dev/issues/145) but with
```
 clubb_c8               =  4.6
```
The goal being to bring increase the cloud and bring the RESTOM closer to zero. 

This similar to #146 but using another set of parameters to retune RESTOM. 

This is based on Adam's comment: 
_If you recall all the way back Jiang Zhu's talk on the Eocene simulations in April ([here](https://drive.google.com/file/d/11bCewR7HdpQeLPlUGPY512CC314H_wEz/view)), C8 is an alternative way to impact the skewness. clubb_gamma hits it really hard because it controls a PDF parameter - the width of the w' distribution. But C8 does it through a more natural way, in that it is the damping coefficient on the prognostic w'^3 equation. Increasing C8 will reduce skewness, which will increase cloud thickness._

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019b

On github:
https://github.com/NCAR/amwg_dev/tree/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019b

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cesm3_cam6_3_058_MOM_e

On github:
hash: https://github.com/NCAR/amwg_dev/commit/8f70c084f85987a02a88df6570c8499a62fbebd9

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/B1850MOM/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019b/atm/](https://webext.cgd.ucar.edu/B1850MOM/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019b/atm/)

**Contacts:**
@cecilehannay, @gustavo-marques, @JulioTBacmeister, @adamrher

**Comment by JulioTBacmeister** [comment]
I think both 18b and 19b are looking good. 18b is a little on the warm side - as expected.  I'd say let them both run another day, and we can reevaluate tomorrow.

![image](https://user-images.githubusercontent.com/9221710/182032792-61af5cdd-1ecc-44dc-8b50-129113c1f97d.png)

**Comment by JulioTBacmeister** [comment]
I'm interested in how c19b evolves. I agree it may be to switch the default c8

**Comment by gustavo-marques** [comment]
Case 18b is looking better than 19b in terms of Lab Sea convection. 
![mld_time_series_lab_sea](https://user-images.githubusercontent.com/11339137/182188495-0802bce3-4ac5-4c84-8c95-78c7774668da.png)

**Comment by JulioTBacmeister** [comment]
We should think about the next steps here.  Looks like convection in 019b died pretty quickly. Correct?  How is convection in 017?

One more timeseries plot:
![image](https://user-images.githubusercontent.com/9221710/182406220-7b2d48b5-be9a-4f75-96e0-89f7c1e06437.png)

**Comment by swrneale** [comment]
I think if 017 is still freezing then we need to take stock. @gustavo-marques remind me. Convection in/near the lab. sea was too strong in the g-cases right, and so there would be no real justification for upping any of the contributions to cvmix?

**Comment by JulioTBacmeister** [comment]
I don't think we have seen that 017 is freezing.  It isn't in Gustavo's plot.

**Comment by gustavo-marques** [comment]
Sorry, I forgot to include 017 in the time series. Below is the updated plot.

![mld_time_series_lab_sea](https://user-images.githubusercontent.com/11339137/182435179-3f0fffcd-4375-498b-98d2-a974eacb1afb.png)

As a reminder, the above time series are area-weighted mixed layer depths (MLDs). The plot below shows see Lab Sea area:

![image](https://user-images.githubusercontent.com/11339137/182435655-4d9c63b3-1de9-4689-9f3b-2bd9004532cb.png)

Focusing on case 017, let's look at the March MLD averaged between years 20-35. 

![image](https://user-images.githubusercontent.com/11339137/182436498-057042cd-2ec4-4c13-a4b6-66fb2286323d.png)

017 is **underestimating** MLDs in the Lab Sea, but it's still convecting. Notice that the most vigorous convection has moved eastward, just east of the southern tip of Greenland. This is also the case for 018b and 019b (see plots below and notice that averages were done between years 15-25 in these cases).

![image](https://user-images.githubusercontent.com/11339137/182454598-c9203d73-8366-4d88-bbd8-339e8cff7d0d.png)

![image](https://user-images.githubusercontent.com/11339137/182454404-17453e04-4774-4e0d-9c62-33463384ff84.png)

@swrneale: yes, convection in the Lab Sea tend to be on the stronger side in G cases and we do not believe there is a justification for tunning cvmix. However, remember that we **do not** have the Estuary Box Model parameterization in MOM at the moment, and this missing phsyics might be contributing to this issue.

**Comment by swrneale** [comment]
So really only 18b has any appreciable deep convection left and that is a bit anaemic.

**Comment by gustavo-marques** [comment]
Updating the MLD plots.

### Area-weighted (Lab Sea) mixed layer time series 

![image](https://user-images.githubusercontent.com/11339137/182672098-b817db9f-0dd2-487d-a97d-426a0b16987f.png)

### March mean MLD (years 25-35)
#### case 009
![image](https://user-images.githubusercontent.com/11339137/182676380-bce8eb89-5db5-46a7-b96d-9f529e5dfaaf.png)

#### case 016 (years 20-40)
![image](https://user-images.githubusercontent.com/11339137/182678933-126055f1-5d60-433c-848e-a6e1b7edf517.png)

#### case 017
![image](https://user-images.githubusercontent.com/11339137/182672407-213055c6-4272-4a43-844a-6e1b1ed5e9f3.png)

#### case 018b
![image](https://user-images.githubusercontent.com/11339137/182672992-db62c155-b89f-4ea8-80fa-ea2239b74b8c.png)

#### case 019b
![image](https://user-images.githubusercontent.com/11339137/182673671-5ed2628d-de46-467f-8bdd-8abdf372455b.png)

### Lab Sea temperature bias (model - WOA18)
#### case 009
![image](https://user-images.githubusercontent.com/11339137/182679315-baadfdec-20c3-49a5-811d-c3ade83faef8.png)

#### case 016
![image](https://user-images.githubusercontent.com/11339137/182680056-2b9435f2-b0ff-4318-b2ff-e5efb027d028.png)

#### case 017
![image](https://user-images.githubusercontent.com/11339137/182679436-fa02f0ea-11a4-47aa-94a9-b1b2fca759a4.png)

#### case 018b
![image](https://user-images.githubusercontent.com/11339137/182679515-6fc20779-4a3a-4575-b8fd-c7f9ecea2061.png)

#### case 019b
![image](https://user-images.githubusercontent.com/11339137/182679587-7e39a14e-cc03-41f8-80e0-53abe2baa763.png)

### Lab Sea salinity bias (model - WOA18)

#### case 009
![image](https://user-images.githubusercontent.com/11339137/182679657-47b7b79c-7173-400d-9003-0e29b7f18427.png)

#### case 016
![image](https://user-images.githubusercontent.com/11339137/182679767-15a9c3f1-4fc7-45f7-9afb-0d23dda8dd5d.png)

#### case 017
![image](https://user-images.githubusercontent.com/11339137/182679820-0d4fe40f-a6ac-4eb2-ac62-e8caf6c660d6.png)

#### case 018b
![image](https://user-images.githubusercontent.com/11339137/182679875-66d573ce-c6da-4694-9bb9-72b6e1fa6f3a.png)

#### case 019b
![image](https://user-images.githubusercontent.com/11339137/182679934-82fc1f0e-ca02-4891-a1e2-6ce8d0c034f9.png)

**Comment by gustavo-marques** [comment]
The Lab Sea in 017 is very fresh near the surface (just like 009), but it's not as cold as 009. It's still convecting a bit, but the Lab Sea might freeze soon.

018b is looking good in terms of T&S biases and MLDs.

**Comment by adamrher** [comment]
@jiang-zhu I'm wondering if you wouldn't mind commenting on the differences between increasing c8 and decreasing gamma. Instead of cranking up c8, could you have instead lowered clubb_gamma to prevent the 6XCO2 runaway greenhouse? I've found that comparing 18b to 17 (clubb_gamma=0.298->0.280) and 19b to 17 (c8=4.2->4.6), the change in SWCF looks very similar, spatially.

![temp_SWCF 18b](https://user-images.githubusercontent.com/29961476/186994990-4d8e0ee1-b678-49b0-ac9d-6a030001af95.png)
![temp_SWCF 19b](https://user-images.githubusercontent.com/29961476/186995003-7815f0f5-9ad3-4dad-a429-30767bc6fb0d.png)

They both look to be doing the same thing, which is expected since they both target skewness, but the patterns are still remarkably similar given the very different ways these two methods effect skewness.

**Comment by jiang-zhu** [comment]
Hi @adamrher clubb_gamma_coef is already low in the 2deg model (0.275). We may need a very low clubb_gamma_coef to tune the Eocene high CO2 simulation, which I didn't do, partly because you once said you usually don't push clubb_gamma_coef to lower than 0.26.

I did test the gamma_coefb (default value=0.32 in parameters_tunable.F90), though. I tried setting gamma_coefb = clubb_gamma_coef = 0.275. The simulated Eocene temperature response to CO2 increase (from 1× to 6×) is not impacted. In contrast, increasing clubb_c8 drastically impacts the temperature sensitivity.
